### PR TITLE
20417-EFT Short Name details

### PR DESCRIPTION
**Pull request by ochiu**
*Issue #:*
https://github.com/bcgov/entity/issues/20417

*Description of changes:*
- update short names detail account linking to support a table for multi linking (use updated endpoint)
- update transaction table to be payment history showing Statement Paid and Funds Received (use updated endpoint)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/pay/eft/ShortNameTransactions.vue [hunk 1]
```diff
@@ -131,14 +145,19 @@ export default defineComponent({
       return `${height}px`
     }
 
+    function isStatementPaid (item: any) {
+      return item?.transactionDescription === 'Statement Paid'
```
Comment (seeker25) [inline auth-web/src/components/pay/eft/ShortNameTransactions.vue:149]: Enum if possible

File: auth-web/src/util/common-util.ts [hunk 2]
```diff
@@ -12,6 +12,11 @@ export default class CommonUtils {
     return value?.startsWith('http')
   }
 
+  // Format payment account display name
```
Comment (seeker25) [inline auth-web/src/util/common-util.ts:15]: no need for the comment? could rename the function to exactly this if needed

**Review by seeker25 — APPROVED** [review]
LGTM

File: auth-web/tests/unit/components/ShortNameAccountLink.spec.ts [hunk 3]
```diff
@@ -1,26 +1,86 @@
...
+        {
+          'accountBranch': 'Test Branch',
+          'accountId': '3202',
+          'accountName': 'Odysseus Chiu',
+          'amountOwing': 51.5,
+          'id': 45,
+          'shortNameId': 2,
+          'statementId': 5407509,
+          'statusCode': 'LINKED',
+          'updatedBy': 'CODDYSEU',
```
Comment (seeker25) [inline auth-web/tests/unit/components/ShortNameAccountLink.spec.ts:53]: I'd nuke this ;)

File: auth-web/tests/unit/components/ShortNameAccountLink.spec.ts [hunk 4]
```diff
@@ -1,26 +1,86 @@
...
+        {
+          'accountBranch': '',
+          'accountId': '3175',
+          'accountName': "Ody's Dev Test 3",
+          'amountOwing': 0.0,
+          'id': 103,
+          'shortNameId': 2,
+          'statementId': 5407406,
+          'statusCode': 'LINKED',
+          'updatedBy': 'CODDYSEU',
```
Comment (seeker25) [inline auth-web/tests/unit/components/ShortNameAccountLink.spec.ts:40]: I'd nuke this ;)

**Review by seeker25 — COMMENTED** [review]
Small points i'd change

File: auth-web/src/components/pay/eft/ShortNameTransactions.vue (hunk 1 above)
Comment (ochiu) [reply auth-web/src/components/pay/eft/ShortNameTransactions.vue]: Added enum.

File: auth-web/src/util/common-util.ts (hunk 2 above)
Comment (ochiu) [reply auth-web/src/util/common-util.ts]: removed comment

File: auth-web/tests/unit/components/ShortNameAccountLink.spec.ts (hunk 3 above)
Comment (ochiu) [reply auth-web/tests/unit/components/ShortNameAccountLink.spec.ts]: Cleaned this up.

File: auth-web/tests/unit/components/ShortNameAccountLink.spec.ts (hunk 4 above)
Comment (ochiu) [reply auth-web/tests/unit/components/ShortNameAccountLink.spec.ts]: Cleaned this up.

**Comment by ochiu** [comment]
Unlinked Short name example:
Unlinked Short name details

Multi Linked short name example:
Multi Linked short name details

**Pull request merged**